src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -304,7 +304,13 @@ CHIP_ERROR Hash_SHA1(const uint8_t * data, const size_t data_length, uint8_t * o
 
 Hash_SHA256_stream::Hash_SHA256_stream() {}
 
-Hash_SHA256_stream::~Hash_SHA256_stream() {}
+Hash_SHA256_stream::~Hash_SHA256_stream()
+{
+    Clear();
```

Comment:
If the purpose of this call is to erase sensitive data, `memset` may not be sufficient as it can be optimized out. `memset_s` might be a safer option.

Reply:
I used the native versions `OPENSSL_cleanse` and `mbedtls_platform_zeroize`